docs/usage/special_features.rst

```diff
@@ -0,0 +1,42 @@
+Special Features
+#################
+
+Re:dash has a lot of very useful features and most of them can be found easily when using the UI. This page features the less well-known ones.
+
+Queries
+========
+It is possible to have filters for query results and visualizations. Thanks to filters, you can restrain the result to a certain or multiple values. Filters are enabled by following a naming convention for columns.
+
+If you want to focus only on a specific value, you will need to alias your column to ``<columnName>::filter``. Here is an example:
+
+.. code:: sql
+
+    select action as "action::filter", count (0) as "actions count"
+    from events
+    group by action
+
+You can see this query and the rendered UI `here <http://demo.redash.io/queries/143/source#table>`_.
+
+If you are interested in multi filters (meaning that you can select multiple values), you will need to alias your column to ``<columnName>::multi-filter``. Here is an example:
+
+.. code:: sql
+
+    select action as "action::multi-filter", count (0) as "actions count"
+    from events
+    group by action
+
+You can see this query and the rendered UI `here <http://demo.redash.io/queries/144/source#table>`_.
+
+Dashboards
+==========
+It is possible to group multiple dashboards in the dashboards menu. To do this, you need to follow a naming convention by using a column (``:``) to separate the dashboard group and the actual dashboard name. For example, if you name 2 dashboards ``Foo: Bar`` and ``Foo: Baz``, they will be grouped under the ``Foo`` namespace in the dropdown menu.
+
+If you've got queries that have some filters and you want to be able to use the filters in a dashboard, you need to enable filters for the dashboard. Currently, there is no way to do this thanks to the UI but you can do it by manually setting the column ``dashboard_filters_enabled`` of the table ``dashboards`` to ``TRUE`` in the Re:dash database.
```

Comment:
1. Worth clarifying that the filters will work. This feature is to enable filters at the dashboard level (that apply to all queries).
2. It can also be done from the admin interface `/admin/dashboard`.